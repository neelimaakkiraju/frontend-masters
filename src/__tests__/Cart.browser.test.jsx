import {render} from "@testing-library/react"
import {expect,test} from "vitest"
import Cart from "../Cart"

test("snapshot when nothing in cart",()=>{
    const {asFragment} = render(<Cart cart={[]}/>)
    expect(asFragment()).toMatchSnapshot()
})