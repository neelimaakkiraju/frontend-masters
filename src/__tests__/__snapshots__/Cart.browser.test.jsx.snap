// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot when nothing in cart 1`] = `
<DocumentFragment>
  <div
    class="cart"
  >
    <h2>
      Cart
    </h2>
    <ul />
    <p>
      Total:â‚¹0.00
    </p>
    <button>
      checkout
    </button>
  </div>
</DocumentFragment>
`;
